cumulus/parachains/integration-tests/emulated/assets/asset-hub-westend/src/tests/mod.rs

```diff
@@ -13,9 +13,44 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+use crate::*;
+
 mod reserve_transfer;
 mod send;
 mod set_xcm_versions;
 mod swap;
 mod teleport;
 mod treasury;
+
+/// Relay Chain sends `Transact` instruction with `force_create_asset` to System Parachain.
+pub fn do_force_create_asset_from_relay_to_system_para(origin_kind: OriginKind) {
+	let asset_owner: AccountId = AssetHubWestendSender::get().into();
+
+	Westend::send_transact_to_parachain(
+		origin_kind,
+		AssetHubWestend::para_id(),
+		AssetHubWestend::force_create_asset_call(ASSET_ID, asset_owner.clone(), true, 1000),
+	);
+
+	// Receive XCM message in Assets Parachain
+	AssetHubWestend::execute_with(|| {
+		type RuntimeEvent = <AssetHubWestend as Chain>::RuntimeEvent;
+
+		AssetHubWestend::assert_dmp_queue_complete(Some(Weight::from_parts(
+			1_019_445_000,
+			200_000,
+		)));
+
+		assert_expected_events!(
+			AssetHubWestend,
+			vec![
+				RuntimeEvent::Assets(pallet_assets::Event::ForceCreated { asset_id, owner }) => {
+					asset_id: *asset_id == ASSET_ID,
+					owner: *owner == asset_owner,
+				},
+			]
+		);
+
```

Comment:
I think this should be moved to `integration-tests-common`. It is not only useful for Westend, it can be used also for Kusama & Polkadot.

Reply:
fixed